hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -64,6 +66,7 @@
 import org.apache.hadoop.hbase.snapshot.UnknownSnapshotException;
 import org.apache.hadoop.hbase.util.Bytes;
 import org.apache.hadoop.hbase.util.Pair;
+import org.apache.hbase.thirdparty.org.apache.commons.collections4.CollectionUtils;
```

Comment:
Is there a nonshaded collections util available? Might be an issue when backporting. Just a nit.